x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.test.tsx

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import * as React from 'react';
-import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { mountWithIntl, nextTick } from 'test_utils/enzyme_helpers';
```

Comment:
Are the changes in this file required for the PR?  Like, the server plugin wasn't completely initialized before, or something?

Reply:
Yeah, it is the last weird test I found. Decided to fix it, because it cause jest tests failing.